DebugEngine/Debuggees/test1/test1.vcxproj

```diff
@@ -86,7 +86,7 @@
       <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
     </ClCompile>
     <Link>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <GenerateDebugInformation>DebugFull</GenerateDebugInformation>

```

Comment:
This is not accepted in VS2013, but the VS2017 compiled Concord plugin crashes when used inside VS2013. So I'm currently still building the release with VS2013 (that does not include the tests, though).

Reply:
I'll change:
`<GenerateDebugInformation>DebugFull</GenerateDebugInformation>`
to
`<GenerateDebugInformation Condition="'$(PlatformToolset)' == 'v120' or  '$(PlatformToolset)' == 'v140'">true</GenerateDebugInformation>`
`<GenerateDebugInformation Condition="'$(PlatformToolset)' != 'v120' and '$(PlatformToolset)' != 'v140'">DebugFull</GenerateDebugInformation>`

I found that this works in VS 2015 and VS 2017. I don't have VS 2013.

Reply:
Looks good, should work for VS2013, too. The usual trouble with handcrafted settings is that you cannot edit them in the project settings dialog, but that's ok IMO.